pkg/kubectl/cmd/testdata/edit/testcase-apply-edit-last-applied-list/3.response

```diff
@@ -0,0 +1,22 @@
+{
+	"kind": "ConfigMap",
+	"apiVersion": "v1",
+	"metadata": {
+		"name": "cm1",
+		"namespace": "edit-test",
+		"selfLink": "/api/v1/namespaces/edit-test/configmaps/cm1",
+		"uid": "b09bffab-e9d7-11e6-8c3b-acbc32c1ca87",
+		"resourceVersion": "1465",
+		"creationTimestamp": "2017-02-03T06:12:07Z",
+		"annotations": {
```

Comment:
Same question goes for all the other stub responses. They look like they were hand edited, not recorded